1_5_5/includes/templates/YOUR_TEMPLATE/templates/tpl_modules_attributes.php

```diff
@@ -77,49 +77,41 @@
          } // END h3_option_comment
          ?>
 
-         <div class="wrapperAttribsOptions" id="<?php echo $options_html_id[$I]; ?>">
-              <h4 class="optionName back"><?php echo $options_name[$I]; ?></h4>
-              <div class="back"><?php echo "\n" . $options_menu[$I]; ?></div>
-              <br class="clearBoth" />
-         </div>
+<div class="wrapperAttribsOptions" id="<?php echo $options_html_id[$i]; ?>">
```

Comment:
Missed an important distinction in this file and is also something that I am going to try to modify, but the code removed included a capital $I which is treated differently than a lower case $i... As modified this will not provide the correct results... Yes I need to change $I to a different variable name.